django/views/i18n.py

```diff
@@ -234,12 +235,31 @@ def get_paths(self, packages):
         # paths of requested packages
         return [os.path.join(app.path, 'locale') for app in app_configs]
 
-    def get_plural(self):
-        plural = None
+    @property
+    def num_plurals(self):
+        """
+        Return the number of plurals for this catalog language.
```

Comment:
Instead of two sentences for these docstrings, I'd say: "Return ...., or 2 if ..."